src/backend/TrafficCourts/Workflow.Service/Consumers/CreateDisputeInDatabaseConsumer.cs

```diff
@@ -62,7 +62,7 @@ public async Task Consume(ConsumeContext<SubmitNoticeOfDispute> context)
             catch (Exception ex)
             {
                 _logger.LogError(ex, "Failed to process message");
-                throw;
+                // dont try again
```

Comment:
I will remove this change from the PR as its not necessary to fix the issue of this PR.  However the reason I had made a change was to avoid putting the message back in the queue -- but I agree about better error handling before just removing it.